### PR TITLE
[cameraInit] Reject input if multiple frames have same frame id

### PR DESCRIPTION
**Pull request by servantftechnicolor**

### CODE REVIEW

File: src/software/pipeline/main_cameraInit.cpp [hunk 1]
```diff
@@ -830,6 +830,23 @@ int aliceVision_main(int argc, char **argv)
       vitem.second->addMetadata("AliceVision:useWhiteBalance", (useInternalWhiteBalance)?"1":"0");
     }
   }
+
+  //Check unique frame id per rig element
+  {
```
Comment (p12tic) [inline src/software/pipeline/main_cameraInit.cpp:835]: I think it makes sense to have this code as a separate function.

Comment (servantftechnicolor) [reply src/software/pipeline/main_cameraInit.cpp:854]: done

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.11)
 project(aliceVision LANGUAGES C CXX)
-
+set(GFLAGS_USE_TARGET_NAMESPACE TRUE)
```
Comment (fabiencastan) [inline CMakeLists.txt:3]: Not related to the PR.
Need a comment to explain this reason for this flag.

Comment (servantftechnicolor) [reply CMakeLists.txt:3]: Junk code.

**Review by mugulmd — APPROVED** [review]

**Pull request merged**